### PR TITLE
fix: update legacy product table content to be an array

### PR DESCRIPTION
**Pull request by samhstn**
# Description

Make content an array instead of a string.

# Checklist

Pull request contains:

- [ ] A new component
- [ ] Component maintenance: improvement / bug fix / etc
- [ ] Component library change: storybook / webpack / etc

Definition of done:

- [ ] Includes theme changes for both Uswitch and money
- [ ] Work has been tested in multiple browsers
- [ ] PR description includes description of change
- [ ] PR description includes screenshot of change
- [ ] If new component, designer has approved screenshot
- [ ] If the change will affect other teams, that team knows about this change
- [ ] If introducing a new component behaviour, added a story to cover that case.

### CODE REVIEW

File: src/compounds/legacy-product-table/src/index.tsx [hunk 1]
```diff
@@ -390,25 +390,30 @@ interface MoreInformationTextProps {
...
-          <p
-            key={key}
-            sx={{
-              fontSize: 'xs',
-              my: 'xxs',
-              mx: '0'
-            }}
-            dangerouslySetInnerHTML={{ __html: line }}
-          ></p>
+          <React.Fragment key={i}>
```
Comment (erkinun) [inline src/compounds/legacy-product-table/src/index.tsx:397]: Why don't we use `<>` or just `<div>`?

**Review by erkinun — APPROVED** [review]
LGTM

File: src/compounds/legacy-product-table/src/index.tsx (hunk 1 above)
Comment (samhstn) [reply src/compounds/legacy-product-table/src/index.tsx:397]: ["the shorthand version does not support the key attribute"](https://thisthat.dev/react-fragment-shorthand-vs-longhand/#:~:text=is%20the%20shorthand%20tag%20for,them%20in%20a%20new%20node.&text=The%20only%20difference%20between%20them,not%20support%20the%20key%20attribute.)

Comment (erkinun) [reply src/compounds/legacy-product-table/src/index.tsx:397]: 👍

**Review by erkinun — DISMISSED** [review]

**Review by erkinun — APPROVED** [review]

**Pull request merged**